README.md

```diff
@@ -50,10 +31,27 @@ South Londoners(Member):
 - Post an opportunity [#2](https://github.com/FACG6/city-in-the-south/issues/2).
 - Update profile [#3](https://github.com/FACG6/city-in-the-south/issues/3).
 
-## TWEET
+## Prototype:
+
+Website prototype is [here](https://www.figma.com/file/Pom80HoRieQ24tSmTzayO4WP/City-of-south?node-id=122%3A474).
+
+## How to use:
+
+- he project is still in builing stage..... In general:
```

Comment:
```suggestion
- This project is still in the builing stage..... In general:
```

Reply:
The refers to the project.
I'll correct typo
thanks you